blog-contribution-guidelines.md

```diff
@@ -0,0 +1,57 @@
+
+**Articles about The Update Framework**
+* Technical content and how-to’s
+* Stories about TUF how TUF helped
+* Use cases and success stories about TUF
+* Industry insight into TUF and TUF’s role in the ecosystem
+* Industry insight about software update system security and how to use TUF
+* Project updates about TUF
+
+
+
+Next, the blog editor will review the content for any grammatical errors, spelling mistakes, technical accuracy, and provide feedback. All feedback and collaboration will take place in the Github Pull Request.
+
+Depending on where your blog falls in the queue, editing and publication dates will vary. Blog content is reviewed and published on a first come, first serve basis. This step should take 5 - 7 business days to complete.
+
+The goal is to publish 1 blog per week and the community commits to a 2 week(s) turnaround end to end. 
+
+**Publishing process**
+* Depending on the quality of the content & edits needed this process should take 5 - 10 business days. 
+* Once the editor and contributor are done with the editing process, the blog should publish when the next available slot opens up. 
+* See editorial calendar
+* Community contributed blogs are published on Tuesday & Thursday on a first come, first serve basis.
+* The blog content is published on the TUF website - [blog](https://theupdateframework.io/).
+
+**Submit your PR**
+Now that you understand the content guidelines and publishing process you are ready to submit your pull request. All blog content must be submitted through this process to be considered for publication. 
+* How to submit your first PR blog in easy steps
+* Step 1: Fork to the [TUF website repo](https://github.com/theupdateframework/theupdateframework.io) on Github
+* Step 2: Clone locally your branch and create a new branch
+* Step 3: Create your blog post in markdown format.
+* Step 4: Add images to the [img/logos folder](https://github.com/theupdateframework/theupdateframework.io/tree/master/static/img/logos)
+* Step 4: Commit your work with a signoff and push to Github
+* Step 5: Create your pull request to upstream repo with your changes
+Filing your first PR? Go here 👉 [for step by step instructions ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+
+**Promoting your blog**
+Finally, once the blog goes live we will promote the content on the TUF social media handle. 
+
+Be sure to provide a headshot and your social media handles so we can tag you in the post. Don’t forget to share your content with your network too! 
+
+Blog content will also be submitted for promotion in the following channels: 
+* [TUF  Slack channel](https://cloud-native.slack.com/archives/C8NMD3QJ3)
+* [TUF Newsletter]()
+* [TUF Twitter handle]() 
+* [TUF LinkedIn page]()
```

Comment:
Hi @Ayush9026 I hope you are doing well, Just a small doubt, does TUF have Twitter and LinkedIn handles?
Thank you👍

Reply:
Hi @h4l0gen ,

I hope you are doing well too! TUF does not currently have Twitter or LinkedIn handles. However, I will ask the maintainers in the slack channel about this and update you accordingly.

Thank you! 👍